packages/datastore-storage-adapter/__tests__/ExpoSQLiteDatabase.test.ts

```diff
@@ -201,7 +200,25 @@ describe('ExpoSQLiteDatabase 2', () => {
 
 			await expect(
 				instance.batchQuery(queryParameterizedStatements)
-			).rejects.toEqual(MOCK_SQL_ERROR);
+			).rejects.toStrictEqual(MOCK_SQL_ERROR);
+		});
+
+		it('should reject promise and return error when the transaction is unsuccessful', async () => {
```

Comment:
Hmm we should turn on linting rules for this but you should never have two tests with the same description. They are testing different things after all.
```suggestion
		it('should reject promise and return error when any SQL execution is unsuccessful', async () => {
```

Reply:
Yes. I have added `jest/no-identical-title` rule to avoid same title for different tests.